### DISCUSSION TITLE
Less than / greater than for iteration field

### DISCUSSION THREAD

**Discussion by knyghty** [community/community#11994] [Projects and Issues] [answered]
The addition of `@current`, `@next`, `@previous` are welcome additions. The next logical step would be the ability to filter by iterations prior to or later than another.

For example, right now I have a planning board, which has the current iteration, all future ones, plus the backlog (no iteration). This requires me to update the filter every iteration so I currently have a filter such as `-iteration:"Iteration 1","Iteration 2","Iteration 3","Iteration 4","Iteration 5","Iteration 6","Iteration 7","Iteration 8"`

Being able to do `-iteration:<@current` would make my life a bit easier.

**Comment by ahosamuel** [comment, marked as answer]
Seems that this is possible now. Have a look here: [Filtering number, date, and iteration fields](https://docs.github.com/en/issues/planning-and-tracking-with-projects/customizing-views-in-your-project/filtering-projects#filtering-number-date-and-iteration-fields)
